src/app/footer/footer.component.css

```diff
@@ -1,60 +1,60 @@
-/*********************************************************** 
-* Copyright (C) 2022 
-* Worktez 
-* 

```

Comment:
What change you have done in this page?

Reply:
There are no changes in this file. It is shown as changed because of the merge conflicts I resolved.